packages/neuron-wallet/src/services/rpc-service.ts

```diff
@@ -8,8 +8,10 @@ export default class RpcService {
   private retryTime: number
   private retryInterval: number
   private rpc: ReturnType<typeof generateRPC>
+  public readonly url: string
```

Comment:
This property has removed on `develop`

Reply:
I just added to ensure that `TransactionCollector` can access this `url` param, this change is due to that the previous RPC url was provided by the `NervinaIndexer`, while now, the indexer's RPC url is `string | undefined`